codalab/worker/bundle_state.py

```diff
@@ -33,19 +36,102 @@ class State(object):
     FINAL_STATES = {READY, FAILED, KILLED}
 
 
+DependencyKey = namedtuple('DependencyKey', 'parent_uuid parent_path')
+Dependency = namedtuple(
+    'Dependency', 'parent_name parent_path parent_uuid child_path child_uuid location'
+)
+
+
+class BundleInfo(object):
+    """
+    Defines the bundle info passed to the worker by the server.
+    Refer to codalab/lib/bundle_util.py#bundle_to_bundle_info() for how the
```

Comment:
Maybe this class needs to be more tightly coupled with that function (ie move this class there or move that function here.